bibliopixel/drivers/APA102.py

```diff
@@ -1,23 +1,19 @@
-from spi_driver_base import DriverSPIBase, ChannelOrder
+from . spi_driver_base import DriverSPIBase, ChannelOrder
 
 
 class DriverAPA102(DriverSPIBase):
     """Main driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
 
-    def __init__(self, num, c_order=ChannelOrder.RGB, use_py_spi=True, dev="/dev/spidev0.0", SPISpeed=2):
-        super(DriverAPA102, self).__init__(num, c_order=c_order,
-                                           use_py_spi=use_py_spi, dev=dev, SPISpeed=SPISpeed)
+    def __init__(self, num, **kwds):
```

Comment:
No, wait, this isn't OK, and I have to figure it out.  :-D

Not in a hurry, right?

And I need to move the test anyway.

On Mon, Jun 20, 2016 at 6:04 PM, adammhaile <notifications@github.com>
wrote:

> In bibliopixel/drivers/APA102.py
> <https://github.com/ManiacalLabs/BiblioPixel/pull/53#discussion_r67775485>
> :
>
> >
> >
> >  class DriverAPA102(DriverSPIBase):
> >      """Main driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
> >
> > -    def __init__(self, num, c_order=ChannelOrder.RGB, use_py_spi=True, dev="/dev/spidev0.0", SPISpeed=2):
> > -        super(DriverAPA102, self).__init__(num, c_order=c_order,
> > -                                           use_py_spi=use_py_spi, dev=dev, SPISpeed=SPISpeed)
> > +    def __init__(self, num, **kwds):
>
> K... I'll pull it in for now, but may want to bring it back later. I
> totally get what you are going for, I just know that a novice would look at
> that and have no idea what the valid inputs were.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/BiblioPixel/pull/53/files/f1413a5dbdcca6ea516504330b5eeec68eb7d502#r67775485>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAPdssVpT1I3jCpRxOKb8Xlz0KcXE1B9ks5qNw5UgaJpZM4I6Iga>
> .
>



-- 
     /t

http://radio.swirly.com - art music radio 24/7 366/1000